features/file_sourcing.feature

```diff
@@ -0,0 +1,62 @@
+Feature: Sourcing tcl files
+
+  Scenario: Display stack trace when there is an error in a sourced file
+    Given a file named "features/test.feature" with:
+      """
+      Feature:
+        Scenario:
+          Given testing file sourcing
+      """
+    And a file named "features/support/env.rb" with:
+      """
+      require 'cucumber/tcl'
+      """
+    And a file named "features/support/env.tcl" with:
```

Comment:
Interesting. That actually answers something else that came up this week!

I'll modify the name of the scenario as suggested.

I've also had another scenario pointed out to me that would fail, so I'll add a test for that and fix the code.

Then I'll add a new issue to look at giving env.tcl special status. I'm assuming that we only need to have it sourced first - if we run in --dry-run mode, env.rb won't be sourced, which means that cucumber-tcl won't be sourced...